webots_ros2_tutorials/webots_ros2_tutorials/slave.py

```diff
@@ -0,0 +1,110 @@
+# Copyright 1996-2020 Soft_illusion.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import rclpy
+from webots_ros2_core.webots_node import WebotsNode
+from std_msgs.msg import Float64
+from geometry_msgs.msg import Twist
+
+
+class service_node_vel(WebotsNode):
+    def __init__(self, args):
+        super().__init__('slave_node', args)
+        # Sensor section
+        self.sensorTimer = self.create_timer(0.001 * self.timestep,
+                                             self.sensor_callback)
+
+        # Enable 3 sensors
+        self.timestep = 16
```

Comment:
Since this member variable is already defined in the parent (WebotsNode) class, it would be safer to use another name:
```suggestion
        self.service_node_vel_timestep = 16
```